docs/commands/intro-to-command-blocks.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Intro to Command Blocks
+category: General
+mention:
+ - BedrockCommands
+ - zheaEvyline
+nav_order: 1
+tags:
+ - info
+---
+
+## Introduction
+
+[Sourced By Bedrock Commands Community Discord](https://discord.gg/SYstTYx5G5)
+
+Command Blocks are special blocks in Minecraft. The same commands (cheats) you type in chat can be run automatically using command blocks and it can be reused without needing to type all over again.
+
+They can only be placed or destroyed by a player with the Operator permission level in gamemode Creative.
+
+## Obtaining
+
+1. Open your world settings.
+2. Under Cheats, toggle "Activate Cheats" setting ON.
+3. Run `/give @s command_block` command in chat.
+
+## Command Block UI
+
+![commandBlockUI](/assets/images/commands/commandBlockUI.png)
+
+## Command Block Types
+
+![impulseCommandBlock](/assets/images/commands/impulseCommandBlock.png) **Impulse** runs the command __once__ each time it is powered.
+
+![chainCommandBlock](/assets/images/commands/chainCommandBlock.png) **Chain** runs the command in a sequence, ie. only after the previous command block it is connecting from was run.
+
+![repeatingCommandBlock](/assets/images/commands/repeatingCommandBlock.png) **Repeat** runs the command every game tick. There are approx 20 ticks per second. A delay can be applied to adjust how often the command is executed (explained [below](/intro-to-command-blocks.html#command-block-tick-delay).)
+
+## Command Block Conditions
+
+**Conditional** command blocks will run the command only if the previous command block it was connecting from had an output that was `true` (successful)
+> Conditional command block states are shown by a small indent into the command block's texture, as shown below:
+> - ![pasteCommandButton](/assets/images/commands/impulseConditionalCommandBlock.png) Impulse Conditional Command Block
+> - ![chainConditionalCommandBlock](/assets/images/commands/chainConditionalCommandBlock.png) Chain Conditional Command Block
+> - ![repeatingConditionalCommandBlock](/assets/images/commands/repeatingConditionalCommandBlock.png) Repeating Conditional Command Block
+
+**Unconditional** command blocks will run the command regardless. Whether the previous command block it was connecting from had an output that was `true` (succesful), `false` (failed) or even if it came with a syntax error the command block will still run the command.
+
+## Command Block Redstone States
+
+**Needs Redstone** command block can only be activated with redstone power. Using buttons, levers, redstone torch etc..
+
+**Always Active** command block will be activated as soon as you close the command block UI.
+
+## Command Block Tick Delay
+
+In this option you may specify how much delay you want there to be before the command block runs the command.
+
+The ticks refer to Minecraft game ticks. A **tick** is simply a unit of measure for time in games. 1 second in real life is approximately 20 game ticks in Minecraft.
+
+:::tip
```

Comment:
Why is this in a tip box?

Reply:
It's not a 100% accurate representation. Just a general idea to have in the back of the head 😅